### DISCUSSION TITLE
Need Help picking the best hosting for NextJS Application with Socket.IO

### DISCUSSION THREAD

**Discussion by mrgreen33gamer** [vercel/next.js#45912] [Help]
In everyone's opinion, what is the best hosting company to do this through?

**Comment by wwwdepot** [comment]
Any server that can host Node.js app.

- Personal use: Cloudflare tunnel + self hosting the app at home
- Low traffic use: any cheap $5 VPS can do it
- Business use: Dockerize the app and run it on any bare metal server or on the cloud with Azure/GC/AWS

--

Alternatively, host Next.js app on Vercel & use Supabase as the backend (with their "[realtime](https://supabase.com/realtime)")
